zip-zcash-grants-fund.rst

```diff
@@ -0,0 +1,330 @@
+     travel expenses, including transportation, hotel and meals allowance.
+
+9. A portion of the Discretionary Budget MAY be allocated to provide reasonable
+   compensation to members of the ZCG Committee. Committee member compensation
+   SHALL be limited to the hours needed to successfully perform their positions
+   and MUST align with the scope and responsibilities of their roles. The
+   allocation and distribution of compensation to committee members SHALL be
+   administered by FPF. The compensation rate and hours for committee members SHALL be determined by
+   the ZF’s Community Advisory Panel or successor process.
+
+FPF SHALL recognize the ZCG allocation of the Grants Fund as a Restricted Fund
+donation under the above constraints (suitably formalized), and keep separate
+accounting of its balance and usage under its `Transparency and Accountability`_
+obligations defined below.
+
+
+Transparency and Accountability
+-------------------------------
+
+Obligations
+~~~~~~~~~~~
+
+ZCG (during and leading to their award
+period) SHALL all accept the obligations in this section.
+
+Ongoing public reporting requirements:
+
+* Quarterly reports, detailing future plans, execution on previous plans, and
+  finances (balances, and spending broken down by major categories).
+* Monthly developer calls, or a brief report, on recent and forthcoming tasks.
+  (Developer calls may be shared.)
+* Annual detailed review of the organization performance and future plans.
+* Annual financial report (IRS Form 990, or substantially similar information).
+
+These reports may be either organization-wide, or restricted to the income,
+expenses, and work associated with the receipt of Zcash Grants Fund.
+
+It is expected that Zcash Community Grants recipients will be focused
+primarily (in their attention and resources) on Zcash. Thus, they MUST
+promptly disclose:
+
+* Any major activity they perform (even if not supported by the Grants Fund) that
+  is not in the interest of the general Zcash ecosystem.
+* Any conflict of interest with the general success of the Zcash ecosystem.
+
+Zcash Community Grants MUST promptly disclose any security or privacy
+risks that may affect users of Zcash (by responsible disclosure under
+confidence to the pertinent developers, where applicable).
+
+All substantial software whose development was funded by the Dev Fund SHOULD
+be released under an Open Source license (as defined by the Open Source
+Initiative [#osd]_), preferably the MIT license.
+
+
+Enforcement
+~~~~~~~~~~~
+
+For Zcash Community Grants recipients, these conditions SHOULD be included in their contract
+with FPF, such that substantial violation, not promptly remedied, will cause
+forfeiture of their grant funds and their return to FPF.
+
+FPF MUST contractually commit to each other to fulfill these
+conditions, and the prescribed use of funds, such that substantial violation,
+not promptly remedied, will permit the other parties to issue a modified version

```

Comment:
Which other parties?

(The developers of Zcash full node software are ECC and ZF, but this needs to be spelled out.)